portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactsP8nInfo.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 The Data-Portability Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataportabilityproject.serviceProviders.google.contacts;
+
+import com.google.common.base.MoreObjects;
+import org.dataportabilityproject.dataModels.PaginationInformation;
+
+public class GoogleContactsP8nInfo implements PaginationInformation {
```

Comment:
I'm wondering if in the new architecture, will we have a generic page token PaginationInformation?

Reply:
Not sure.  Should I just keep going with this until we know more?